serverless-http.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(app, opts) {
   const options = Object.assign({}, defaultOptions, opts);
 
   return (evt, ctx, callback) => {
+    if (options.callbackWaitsForEmptyEventLoop != null) {
```

Comment:
I am using this as a backend service that connects to a database and a redis instance. Such open connections will keep the event-loop going forever.